src/components/content-tab-overview.tsx

```diff
@@ -225,24 +227,53 @@ export function ContentTabOverview( { selectedSite }: ContentTabOverviewProps )
 	} = useThemeDetails();
 
 	const loading = loadingThemeDetails || loadingThumbnails || initialLoading;
+	const siteRunning = selectedSite.running;
 
 	return (
 		<div className="p-8 flex max-w-3xl">
 			<div className="w-52 ltr:mr-8 rtl:ml-8 flex-col justify-start items-start gap-8">
 				<h2 className="mb-3 a8c-subtitle-small">{ __( 'Theme' ) }</h2>
 				<div
 					className={ cx(
-						'w-full min-h-40 max-h-60 overflow-hidden rounded-sm border border-a8c-gray-5 bg-a8c-gray-0 mb-2 flex items-center justify-center',
-						loading && skeletonBg,
-						isThumbnailError && 'border-none'
+						'w-full min-h-40 max-h-64 rounded-sm border border-a8c-gray-5 bg-a8c-gray-0 mb-2 flex justify-center',
+						loading && `h-64 ${ skeletonBg }`,
+						isThumbnailError && 'border-none',
+						siteRunning && 'hover:border-a8c-blueberry duration-300'
```

Comment:
Following the conditions used to render the "Open site" button, we could use the same to enable the hover styles:

```suggestion
						! loading && siteRunning && 'hover:border-a8c-blueberry duration-300'
```

Reply:
Fixed: https://github.com/Automattic/studio/pull/357/commits/63f31dbde34fe590473df97f72d3e0e97dfd26d8